### PR TITLE
fetch last 3 blog posts from ghost api

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -26,6 +26,14 @@ module.exports = {
           include: /\.inline\.svg$/
         }
       }
+    },
+    {
+      resolve: 'gatsby-source-ghost',
+      options: {
+        apiUrl: 'https://blog.hermez.io',
+        contentApiKey: 'a0bbfb0931ce4534811bee1acf',
```
Comment (AlbertoElias) [inline gatsby-config.js:34]: This might need to be an env variable

**Review by AlbertoElias — CHANGES_REQUESTED** [review]
small thing

File: gatsby-config.js (hunk 1 above)
Comment (elias-garcia) [reply gatsby-config.js:34]: Yep! This needs to be an env var. Make sure you rewrite the git history when doing it @mmilenkovic, so the `contentApiKey` can't be seen at any old commit.

Comment (mmilenkovic) [reply gatsby-config.js:34]: I will delete this branch and do commit in a new one, so we don't need to worry about history.

Comment (mmilenkovic) [reply gatsby-config.js:34]: contentApiKey regenerated. Secret updated. Security restored. 🥳

**Pull request closed without merging**